apns2/client.py

```diff
@@ -92,9 +92,9 @@ def send_notification_batch(
         if the token was sent successfully, or the string returned by APNs in the 'reason' field of
         the response, if the token generated an error.
         '''
-        token_iterator = iter(tokens)
+        notification_iterator = iter(notifications)
         try:
-            next_token = next(token_iterator)
+            next_notification = next(notification_iterator)
```

Comment:
I'm not sure why that would be better, and anyway I think we should deal with it in a separate thread since it's unrelated to this PR.